app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1>Task List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <th class="center">
+      Mark Done
+    </th>
+    <th class="center">
+      Details
+    </th>
+    <th class="center">
+      Delete
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
```

Comment:
Err... model? You mean 

```ruby
class Task < ActiveRecord::Base
  def completed?
    task.completed_date.present?
  end
end
```

Because the CSS I feel should live in the views. That's where rails uses helpers (which you may not have discussed yet) — to provide encapsulation of view logic without polluting the model or controller.